app/src/main/res/layout/list_contact_doctor.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="wrap_content"
+  android:background="?attr/selectableItemBackground"
+  android:clickable="true"
+  android:focusable="true"
+  android:paddingStart="@dimen/spacing_24"
+  android:paddingTop="@dimen/spacing_8"
+  android:paddingEnd="@dimen/spacing_24"
+  android:paddingBottom="@dimen/spacing_8">
+
+  <TextView
+    android:id="@+id/doctorNameTextView"
+    android:layout_width="0dp"
+    android:layout_height="wrap_content"
+    android:layout_marginEnd="@dimen/spacing_16"
+    android:textAppearance="@style/Clinic.V2.TextAppearance.Body0Left.Emphasis"
+    app:layout_constraintBottom_toTopOf="@id/doctorNumberTextView"
+    app:layout_constraintEnd_toStartOf="@id/whatsAppDoctorImageView"
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintTop_toTopOf="parent"
+    tools:text="Dr Sunil Dhar" />
+
+  <TextView
+    android:id="@+id/doctorNumberTextView"
+    android:layout_width="0dp"
+    android:layout_height="wrap_content"
+    android:textAppearance="@style/Clinic.V2.TextAppearance.Body2Left.Numeric.Grey1"
+    app:layout_constraintBottom_toBottomOf="parent"
+    app:layout_constraintEnd_toEndOf="@id/doctorNameTextView"
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintTop_toBottomOf="@id/doctorNameTextView"
+    tools:text="+911111111111" />
+
+  <ImageView
+    android:id="@+id/whatsAppDoctorImageView"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_marginEnd="@dimen/spacing_16"
+    android:background="?attr/selectableItemBackgroundBorderless"
+    android:clickable="true"
+    android:focusable="true"
+    android:src="@drawable/ic_whatsapp_circular"
```

Comment:
Shouldn't we be using `app:srcCompat` now?

Reply:
OMG yes, I completely forgot. Maybe we should add a lint rule for this 😂